### ISSUE TITLE
[IDEA] Patch instead of replace

### ISSUE THREAD

**Issue by KaidenP** [bambusoft/sentora-paranoid#6]
I noticed that your script just overwrites some of the existing config files, and I was thinking, wouldn't it be a good idea to patch the files, rather than replacing them?

This way, it will keep user changes to the config files.

Of course, I would test to see if there are user changes, and ask if you would like to attempt to patch it, or simply just replace it in case of configuration incompatibilities.

Is this a good idea?

**Comment by bambusoft** [comment]
Yes, it is a good idea for devolopment and testing purposes, but consider that the sentora & sentora-paranoid scripts are designed to install in a clean environment and be runned just once at the first time,..if you want to change it to use patch go ahead, but i do not know how many validations must be included to work properly
